muse/ui/src/components/OverlayLoader.tsx

```diff
@@ -5,9 +5,9 @@ import { Typography } from './Typography';
 export const OverlayLoader = () => {
   return (
     <Dialog open={true}>
-      <DialogTitle>Setting up load balancer</DialogTitle>
+      <DialogTitle>Setting up servers to handle your requests</DialogTitle>
       <DialogContent>
-        <Typography colorI="primary">This allows Muse to handle multiple requests</Typography>
+        <Typography colorI="primary">This allows Muse to generate images at scale</Typography>
```

Comment:
"generate images at scale" sounds like a single image is very large. cc: @RobertLaurella